src/pages/workspace/WorkspaceNewRoomPage.js

```diff
@@ -143,59 +137,56 @@ class WorkspaceNewRoomPage extends React.Component {
             return null;
         }
 
+        const workspaceOptions = _.filter(this.props.workspaceOptions, policy => !!policy);
+
         const visibilityOptions = _.map(_.values(CONST.REPORT.VISIBILITY), visibilityOption => ({
             label: this.props.translate(`newRoomPage.visibilityOptions.${visibilityOption}`),
             value: visibilityOption,
             description: this.props.translate(`newRoomPage.${visibilityOption}Description`),
         }));
 
         return (
-            <ScreenWrapper onTransitionEnd={this.focusRoomNameInput}>
+            <ScreenWrapper>
                 <HeaderWithCloseButton
                     title={this.props.translate('newRoomPage.newRoom')}
                     onCloseButtonPress={() => Navigation.dismissModal()}
                 />
-                <ScrollView style={styles.flex1} contentContainerStyle={styles.p5}>
+                <Form
+                    formID={ONYXKEYS.FORMS.NEW_ROOM_FORM}
+                    submitButtonText={this.props.translate('newRoomPage.createRoom')}
+                    style={[styles.mh5, styles.mt5, styles.flexGrow1]}
+                    validate={this.validate}
+                    onSubmit={this.submit}
+                    enabledWhenOffline
+                >
                     <View style={styles.mb5}>
                         <RoomNameInput
-                            ref={el => this.roomNameInputRef = el}
+                            inputID="roomName"
+                            defaultValue={CONST.POLICY.ROOM_PREFIX}
+                            autoFocus
                             policyID={this.state.policyID}
```

Comment:
@fedirjh We should update the `state.policyID`, probably in the validate function.

Reply:
@mollfpr I think we should remove it completely , `state.policyID` is being passed to `RoomNameInput` but it was not used , not even defined in `roomNameInputPropTypes`

Reply:
@fedirjh You're right! But just to make sure that removed policyID prop doesn't change anything.

Reply:
@mollfpr I checked that , it was used to check for room duplication name before but since then it was refactored and now the validation is done in the `validate` function , check [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1671550870127079?thread_ts=1670710080.656329&cid=C01GTK53T8Q)